src/main/java/uk/gov/di/ipv/cri/common/library/service/SessionService.java

```diff
@@ -20,6 +20,7 @@
 public class SessionService {
     private static final String SESSION_TABLE_PARAM_NAME = "SessionTableName";
     private static final String GOVUK_SIGNIN_JOURNEY_ID = "govuk_signin_journey_id";
+    private static final String VERIFICATION_SCORE = "verification_score";
```

Comment:
Maybe we could call this something like `requested_verification_score`? There's been som confusion about how it's possible to have a verification score before a user has finished verifying and I think this would help with that